helix-core/src/main/java/org/apache/helix/controller/stages/BestPossibleStateCalcStage.java

```diff
@@ -358,10 +357,10 @@ private boolean validateInstancesUnableToAcceptOnlineReplicasLimit(final Resourc
     if (maxInstancesUnableToAcceptOnlineReplicas >= 0) {
       // Instead of only checking the offline instances, we consider how many instances in the cluster
       // are not assignable and live. This is because some instances may be online but have an unassignable
-      // InstanceOperation such as EVACUATE, DISABLE, or UNKNOWN. We will exclude SWAP_IN instances from
+      // InstanceOperation such as EVACUATE, and DISABLE. We will exclude SWAP_IN and UNKNOWN instances from
       // they should not account against the capacity of the cluster.
       int instancesUnableToAcceptOnlineReplicas = cache.getInstanceConfigMap().entrySet().stream()
```

Comment:
Name it as instancesUnableToAcceptLiveReplicas? Online could lead to specific online/offline state model.

Reply:
Not relevant with latest updates